SQF/dayz_code/stringtable.xml

```diff
@@ -23713,23 +23716,25 @@
 		<Key ID="STR_EPOCH_PLAYER_121">
 			<English>Packing %1, move from this position to cancel within 5 seconds.</English>
 			<German>%1 wird gepackt. Bewege dich in den nächsten 5 Sekunden um den Vorgang abzubrechen.</German>
-			<Russian>Упаковывание %1, сдвиньтесь с этой позиции в течение 5 секунд для отмены.</Russian>
+			<Russian>Упаковывание: %1. Для отмены сдвиньтесь с этой позиции в течение 5 секунд.</Russian>
 			<Dutch>Bezig met inpakken van %1, beweeg binnen 5 seconden om te annuleren.</Dutch>
 			<French>Emballage de %1, déplacez vous pour annuler dans moins de 5 secondes.</French>
 			<Czech>Balení %1. Pro zrušení se do 5 sekund pohněte.</Czech>
 		</Key>
 		<Key ID="STR_EPOCH_PLAYER_122">
 			<English>Packing %1 canceled.</English>
 			<German>Packen des %1 wurde abgebrochen.</German>
-			<Russian>Упаковывание %1 отменено.</Russian>
+			<!-- <Russian>Упаковывание %1 отменено.</Russian> -->
+			<Russian>Отменено.</Russian>
```

Comment:
Is this correct?

Reply:
Yes, this is correct. I will still think about translating some strings.